internal/hardware/hardware_test.go

```diff
@@ -24,6 +24,7 @@ var _ = Describe("Hardware", func() {
 
```

Comment:
It would be great to add a test or two for something else than `Not(BeNil())`

Reply:
yes. but for that, we need to find a way to run tests as root.  Nevertheless, we need needs this for `mount` feature also.

Reply:
Can't you mock the `getDevices` function? Because IMO, testing if the `getDevices` itself returns the correct list of devices on the host belong in the e2e tests rather than UT

Reply:
tu t'arrete pas!! cest la 2eme fois déjà :)